### ISSUE TITLE
feat: increase color contrast of code comments

### ISSUE THREAD

**Issue by chrisvaillancourt** [chrisvaillancourt/website#103]
Code comments in blog posts have insufficient color contrast and result in lower lighthouse scores.

**Comment by chrisvaillancourt** [comment]
Need to look more into this. Shiki doesn't have an easy hook into the code comment style.

**Comment by chrisvaillancourt** [comment]
Look into ShikiTransformer support which was introduced in [astro v4.11](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md#4110).

> Adds ShikiTransformer support to the `<Code />` component with a new transformers prop.
Note that transformers only applies classes and you must provide your own CSS rules to target the elements of your code block.

**Comment by chrisvaillancourt** [comment]
See if we can use [css variables described here](https://docs.astro.build/en/guides/upgrade-to/v5/#renamed-shiki-css-variables-theme-color-token-names) to customize color:

`--astro-code-foreground` and `--astro-code-background`

**Comment by chrisvaillancourt** [comment]
also related: https://docs.astro.build/en/guides/syntax-highlighting/
